### PR TITLE
Align with r11 header end

### PR DESCRIPTION
**Pull request by michal-josef-spacek**

### CODE REVIEW

File: src/header_variables_r11.spec [hunk 1]
```diff
@@ -323,3 +323,7 @@
     return 0;
   FIELD_CAST (VISRETAIN, RS, B, 70); //ok
 
+  /* TODO Unknown 18 bytes */
+  DEBUG_HERE //0x6bd-0x6ce
+  dat->byte = 0x6cf;
```
Comment (rurban) [inline src/header_variables_r11.spec:328]: I think, we should better store the unknown slack in some buffer. For encode

**Review by rurban — CHANGES_REQUESTED** [review]

File: src/header_variables_r11.spec (hunk 1 above)
Comment (michal-josef-spacek) [reply src/header_variables_r11.spec:328]: Hm, this is probably not part of header.
I have two versions of DWG, which have 204 and 205 header variables. And difference is VISRETAIN value.

Comment (rurban) [reply src/header_variables_r11.spec:328]: I suspect it's a similar kind of sentinel, some magic, unique bytes after the headers, and before the entities

**Comment by rurban** [comment]
Replaced by something better

**Comment by michal-josef-spacek** [comment]
@rurban ah, interesting. Small fix for this https://github.com/LibreDWG/libredwg/pull/464

**Pull request closed without merging**